vignettes/qenv.Rmd

```diff
@@ -147,11 +145,10 @@ These functions can be seamlessly integrated into `shiny` applications to produc
 
 When employing a `qenv` to evaluate code, should an error occur, an object of type `qenv.error` is generated. This object can be utilized wherever a `qenv` object is used, alleviating the need for code alterations to handle these errors. Select the `error_option` in the example below to witness `qenv` error handling in action.
 
-```{r}
+```{r, eval=FALSE}
```

Comment:
if it's about `shiny` you can do `eval = requireNamespace("shiny")` so that it would eval if one would call it interactively

Reply:
@pawelru 

But isn't with `eval=FALSE`, you can still run the code interactively?
I don't see the point in evaluating this shiny code for vignette creation since there's no output anyway. 

I also don't see any difference in behavior between `eval=FALSE` and `eval=requireNamespace("shiny")` if the user doesn't have `shiny` installed and they run the chunk interactively. I tested this, and in both cases, an error occurs stating `Error in library(shiny): there is no package called 'shiny'`.

Reply:
Sorry - in my previous message I meant "if one has `shiny` preinstalled" instead of "inside interactive run". 

You are right about running the code by the end user. Regardless of the `eval` parameter, code chunks can be freerly copied and evaluated inside user session.

Package vignettes needs to be built, then installed and only then these pre-built files can be browsed by the end user. It's quite likely that `shiny` will be there during the build process hence this code chunk would be actually evaluated. We do want to have as much code auto-tested as possible. I'm afraid that with `eval = FALSE` we are going to hard-exclude too much.

Reply:
> We do want to have as much code auto-tested as possible.

Yes, I agree with this. 👍🏼